exonum/src/helpers/user_agent.rs

```diff
@@ -48,4 +67,16 @@ mod tests {
             assert!(!val.is_empty());
         }
     }
+
+    #[test]
+    fn check_exonum_versions() {
+        let exonum_version = exonum_version().unwrap();
+        assert!(exonum_version.contains("exonum"));
```

Comment:
What's the point of including "exonum" prefix into `exonum_version()`? Same question regarding the "rust" prefix for `rust_version`.